src/documentation/examples/TeaserSection.tsx

```diff
@@ -14,7 +14,7 @@ export default class App extends Vue {
           kicker="kicker"
           text="test1"
           buttonText="click me"
-          handleButtonClick={() => {
+          onClick={() => {
```

Comment:
Please change the event to `onButtonClick`. (Although i've told you otherwise before)